packages/mui-utils/src/useEventCallback/useEventCallback.ts

```diff
@@ -4,7 +4,13 @@ import useEnhancedEffect from '../useEnhancedEffect';
 /**
  * https://github.com/facebook/react/issues/14099#issuecomment-440013892
  */
-export default function useEventCallback<Args extends unknown[], Return>(
+function useEventCallback<T extends (...args: any[]) => any = (...args: unknown[]) => any>(
```

Comment:
Could you rename `T` to something like `Func`? We're trying to get away from one letter generic parameters.
Also, would it still work if the default return type was also `unknown`?